vagrant/Puppetfile

```diff
@@ -0,0 +1,15 @@
+# mod 'golja-gnupg',  '1.2.3'
+mod 'gnupg', :git => 'https://github.com/flepoutre/golja-gnupg.git', :branch => 'master'
+mod 'ipcrm-echo', '0.1.8'
+mod 'puppet-hdm', :git => 'https://github.com/voxpupuli/puppet-hdm.git', :branch => 'fix_rvm'
```

Comment:
this will be changed, or is this a static branch?

Reply:
This is now fixed, as the feature branch is merged into master